open-company/open-company-web
# Email wall restyle

Card: https://trello.com/c/RpI1os2R

To test:
- open an anonymous session
- visit /email-required?e=your-email@here.com
- [x] do you see the new style? Good
